code/game/machinery/recycler.dm

```diff
@@ -116,9 +116,12 @@ var/const/SAFETY_COOLDOWN = 100
 
 	for(var/i in to_eat)
 		var/atom/movable/AM = i
-		if(isliving(AM))
+		var/obj/item/bodypart/head/as_head = AM
+		var/brain_holder = istype(AM, /obj/item/organ/brain) || as_head.brain

```

Comment:
there is NO guarantee here that ```as_head``` will he a head.
meaning there's no guarantee it'll have a ```.brain``` var.
meaning it'll runtime.